rampwf/utils/command_line.py

```diff
@@ -108,3 +109,59 @@ def ramp_convert_notebook():
     parser = create_ramp_test_notebook_parser()
     args = parser.parse_args()
     convert_notebook(ramp_kit_dir=args.ramp_kit_dir)
+
+
+def create_ramp_blend_submissions_parser():
+    import argparse
+    parser = argparse.ArgumentParser(
+        prog='ramp_blend_submissions',
+        description='Blend several submissions.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--ramp_kit_dir',
+                        default='.',
+                        type=str,
+                        help='Root directory of the ramp-kit to test.')
+    parser.add_argument('--ramp_data_dir',
+                        default='.',
+                        type=str,
+                        help='Directory containing the data. This directory'
+                        ' should contain a "data" folder.')
+    parser.add_argument('--submissions',
+                        default='ALL',
+                        type=str,
+                        help='The submissions to blend. They should be located'
+                        ' in the "submissions" folder of the starting kit. If'
+                        ' "ALL", all submissions in the directory will be'
+                        ' blended.')
+    parser.add_argument('--save-y-preds', dest='save_y_preds',
+                        action='store_true',
+                        help='Specify this flag to save predictions '
+                             'after training.')
```

Comment:
`save-y-preds` => `save-predictions` ?
`save_y_preds` => `save_predictions` ?

Reply:
Discutable. One thing would be to put the saving routine in the Predictions classes in which case your naming makes more sense. Now that I write it, it makes more sense.